lambda-notes/package-lock.json

```diff
@@ -0,0 +1,302 @@
+{
```

Comment:
Consider doing the following:
(1) Delete Node Modules
(2) Delete package-lock.json
(3) Delete yarn.lock
(4) Pick Yarn or NPM
(5) Yarn install or NPM install
(6) Do not use both again in a project, pick one or the other for better results :)

Reply:
In the past I found that Yarn and NPM packages might conflict with each other.